Errors while building for PHP-5.3

System: Ubuntu 14.04 x86
PHP: 5.3.29
Previous version 1.13.2 builded without errors.

Build output:
```
libtool: compile:  cc -I. -I/home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src -DPHP_ATOM_INC -I/home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src/inclu
de -I/home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src/main -I/home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src -I/usr/include/php53 -I/usr/include/ph
p53/main -I/usr/include/php53/TSRM -I/usr/include/php53/Zend -I/usr/include/php53/ext -I/usr/include/php53/ext/date/lib -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -I/home/sergey/debuild-
php-modules/php-mod-zip/php-mod-zip-1.13.3/src/lib -I/home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src/php5 -DHAVE_CONFIG_H -g -O2 -fstack-protector --param=ssp-buffer-
size=4 -Wformat -Werror=format-security -O2 -pipe -D_FORTIFY_SOURCE=2 -c /home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src/php5/php_zip.c  -fPIC -DPIC -o php5/.libs/php
_zip.o
libtool: compile:  cc -I. -I/home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src -DPHP_ATOM_INC -I/home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src/inclu
de -I/home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src/main -I/home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src -I/usr/include/php53 -I/usr/include/ph
p53/main -I/usr/include/php53/TSRM -I/usr/include/php53/Zend -I/usr/include/php53/ext -I/usr/include/php53/ext/date/lib -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -I/home/sergey/debuild-
php-modules/php-mod-zip/php-mod-zip-1.13.3/src/lib -I/home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src/php5 -DHAVE_CONFIG_H -g -O2 -fstack-protector --param=ssp-buffer-
size=4 -Wformat -Werror=format-security -O2 -pipe -D_FORTIFY_SOURCE=2 -c /home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src/php5/zip_stream.c  -fPIC -DPIC -o php5/.libs/
zip_stream.o
/home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src/php5/php_zip.c: In function 'php_zip_add_from_pattern':
/home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src/php5/php_zip.c:1835:7: warning: format '%ld' expects argument of type 'long int', but argument 5 has type 'size_t' [-W
format=]
       MAXPATHLEN - 1, (add_path_len + file_stripped_len));
       ^
/home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src/php5/php_zip.c: In function 'zm_startup_zip':
/home/sergey/debuild-php-modules/php-mod-zip/php-mod-zip-1.13.3/src/php5/php_zip.c:3159:21: error: 'zend_object_handlers' has no member named 'get_gc'
  zip_object_handlers.get_gc          = php_zip_get_gc;
                     ^
make[1]: *** [php5/php_zip.lo] Error 1
```

--- a/php5/php_zip.c
+++ b/php5/php_zip.c
@@ -1060,6 +1060,7 @@ static int php_zip_has_property(zval *object, zval *member, int type KEY_ARG_DC 
 }
 /* }}} */
 
+#if PHP_VERSION_ID >= 50400
 static HashTable *php_zip_get_gc(zval *object, zval ***gc_data, int *gc_data_count TSRMLS_DC) /* {{{ */
 {
 	*gc_data = NULL;
@@ -1067,6 +1068,7 @@ static HashTable *php_zip_get_gc(zval *object, zval ***gc_data, int *gc_data_cou
 	return zend_std_get_properties(object TSRMLS_CC);
 }
 /* }}} */
+#endif
 
 static HashTable *php_zip_get_properties(zval *object TSRMLS_DC)/* {{{ */
 {
@@ -3156,7 +3158,9 @@ static PHP_MINIT_FUNCTION(zip)
 	zip_object_handlers.clone_obj		= NULL;
 	zip_object_handlers.get_property_ptr_ptr = php_zip_get_property_ptr_ptr;
 
+#if PHP_VERSION_ID >= 50400
 	zip_object_handlers.get_gc          = php_zip_get_gc;
+#endif
 	zip_object_handlers.get_properties = php_zip_get_properties;
 	zip_object_handlers.read_property	= php_zip_read_property;
 	zip_object_handlers.has_property	= php_zip_has_property;

